test/utils.js

```diff
@@ -14,6 +14,33 @@ describe('utils', function() {
     });
   });
 
+  describe('setLogLevel', function() {
+    afterEach(() => {
+      utils.setLogLevel('WARN');
+    });
+
+    it('can set log level to DISABLE', function() {
+      utils.setLogLevel('DISABLE');
+      assert.strictEqual(utils.getLogLevel(), 0);
```

Comment:
nit, assert on logLevels.DISABLE

Reply:
Thanks updated.

This API is kind of dumb. `setLevel` takes a string and `getLevel` returns this enumerated value. :-/